data/map planets.txt

```diff
@@ -3526,6 +3532,7 @@ planet Redwatch
 	attributes hai station uninhabited
 	landscape land/station6
 	description `This station was mothballed sometime in the last sixty thousand years. The docking area has an atmosphere, and a status panel indicates that the station's generators are functional but shut down, with standby mechanisms running on solar power alone.`
+	description `	In contrast to the Hai's normally durable engineering, this station appears to have sustained substantial damage since it was abandoned. Given its proximity to Unfettered space, the station's scars are most likely a reflection of the current war taking place between the Hai and their northern brethren.`
```

Comment:
```suggestion
	description `	Despite the Hai's notably durable engineering, this station appears to have sustained substantial damage since it was abandoned. Given its proximity to Unfettered space, the station's scars are most likely a reflection of the current war taking place between the Hai and their northern brethren.`
```
Soft suggestion. I think it makes a bit more sense put this way. There's something odd about contrasting the engineering with the damage. I think it's more natural to say that even the Hai's engineering talents could not prevent this station from receiving damage.